oarepo_vocabularies/records/api.py

```diff
@@ -21,9 +21,11 @@
 class SpecialVocabulariesAwarePIDFieldContext(VocabularyPIDFieldContext):
     def resolve(self, pid_value):
         if isinstance(pid_value, str):
-            pid_value = (self._type_id, pid_value)
+            pid_type = self._type_id
+            item_id = pid_value
+        else:
+            pid_type, item_id = pid_value
 
-        pid_type, item_id = pid_value
         specialized_service = current_oarepo_vocabularies.get_specialized_service(pid_type)
         if not specialized_service:
             return super().resolve(pid_value)
```

Comment:
passing original pid value here, not the composed one in case of original pid_value is str